internal/templates/cartogramui.html

```diff
@@ -189,63 +223,69 @@ <h4>Confirm your data are appropriate for a cartogram</h4>
 
       <div id="cartogram">
         <div class="row" id="cartogram-row">
-
           <div class="col-md-6" >
-              
               <div style="height: 40px">
                   <h4 class="mb-0">Equal-Area Map</h4>
                   <!--<p id="tooltip">&nbsp;</p>-->
-                </div>
-               
-
-               <div id = "map-area" style="margin-top: 20px"></div>
+              </div>
 
-              <div class="col-12">
-                  <svg width="300" height="90" id="map-area-legend"></svg>
-                  <img src="/static/creativecommons_icon.svg" width="80" alt="Creative Commons Icon" style="position: absolute; top: 10px; margin-right: 0px">
+              <div id = "map-area" style="margin-top: 20px" data-grid-visibility="off"></div>
+              <div style="padding-left: 0; padding-top: 10px" class="col-12"> <!-- padding-top to add spacing between mapSVG and legend -->
+                <svg width="375" height="90" id="map-area-legend" data-legend-type="static" data-current-grid-path="gridA"></svg>
+                <img src="/static/creativecommons_icon.svg" width="80" alt="Creative Commons Icon" style="position: absolute; top: 10px; margin-right: 0px">
               </div>
 
-               <p style="margin-top: 20px"><a class="btn btn-primary" href="" id="map-download">Download</a></p>
+              <div class="d-flex">
+                <p style="margin-top: 20px; color: white; cursor: pointer" class="d-inline-block"><a class="btn btn-primary btn-customise"
+                     id="map-customise">Customise</a></p>
+                <p style="margin-top: 20px; margin-left: 10px"><a class="btn btn-primary" href="" id="map-download">Download</a></p>
+              </div>
 
+              <div style="display: none" class="customise-popup" id="map-customise-popup">
+                <form class="customise-container">
+                  <input type="checkbox" class="checkbox-customise" id="gridline-toggle-map" name="gridline" value="false">
```

Comment:
```suggestion
                  <input type="checkbox" class="checkbox-customise" id="gridline-toggle-map" name="gridline" value="false" autocomplete="off">
```
Without `autocomplete="off"`, Firefox will remember the checkbox setting when you refresh the page, which will cause the gridline option to appear enabled even though it isn't. Please ensure that all the checkboxes in both customize pop-ups have this `autocomplete="off"` attribute to fix this bug.

Reply:
Thank you for reviewing. I have added the `autocomplete="off"` attribute to all instances of checkboxes in a new commit.